java/ehcache/Portfile

```diff
@@ -1,11 +1,16 @@
 PortSystem 1.0
+PortGroup			java 1.0
+PortGroup			github 1.0
 
-name				ehcache
-version				1.1
+github.setup 			ehcache ehcache3 3.8.1 v
+github.tarball_from releases
+name 				ehcache
+distname 			${name}-${version}
 
 categories			java
 maintainers			nomaintainer
 platforms			darwin
+license				Apache-2
```

Comment:
Please make sure that you use consistent whitespacing (spaces only, no tabs, align to 20 spaces) throughout the file. If the original uses tabs, create a separate commit that first fixes whitespacing, and then another commit with functional changes.

Reply:
Hi, Thanks for the suggestions. I'll rectify it soon.

Reply:
Hi, I've added the whitespaces, but the final alignment doesn't seem to serve the purpose. You may kindly find the same in my commit and revert. I'm trying out a way of ensuring a fine alignment. Any advice in this regard is always highly appreciated.

Reply:
You need to use spaces instead of tabs.

Reply:
Hi, Thanks to both @mojca and @mf2k for pointing it out. I've corrected and incorporated the spacing, hopefully as required. Please revert in case of any issue.